Tools/ssl/multissltests.py

```diff
@@ -535,17 +573,38 @@ def main():
         # check for configure and run make
         configure_make()
 
-    # download and register builder
+    logging.basicConfig(
+        level=logging.DEBUG if args.debug else logging.INFO,
+        format="*** %(levelname)s %(message)s"
+    )
+
+    versions = []
+    ssl_libs = AbstractBuilder.__subclasses__()
```

Comment:
Please, don't use this. It's harder to maintain if we do it like this. Just hardcode the classes.